source/Cosmos.System2/Graphics/FullScreenCanvas.cs

```diff
@@ -1,12 +1,23 @@
 //#define COSMOSDEBUG
 using Cosmos.HAL;
+using sys = System;
```

Comment:
Not sure about if this addition can break anything.
would love to hear your thoughts.

Reply:
Again I dont really understand why you are adding this?

Reply:
Latest master has this line removed (no System methods are used, everything is in-house so to speak)
Makes no difference whether it's added or removed so it's been left out as it's unlikely that a Console or other System namespace method would be used with the Graphics subsystem (for now anyway...)

Can easily be added at a later stage if needed though.

Reply:
@Arawn-Davies The reason for this line was for the docs.
I've added it because Cosmos.System shadowed System, and therefore exceptions types were unknown.
Please check if the compiler idenify the exceptions type.
I believe that, for example, in line 62 the exception "sys.ArgumentOutOfRangeException" is of unknown type now.